source/common/logger/ur_sinks.hpp

```diff
@@ -48,26 +49,41 @@ class Sink {
         flush_level = logger::Level::ERR;
     }
 
+    void flush_buffer() {
+        std::scoped_lock<std::mutex> output_lock(output_mutex);
+        get_buffer().sync();
```

Comment:
does this sync/flush do anything on stringstream?

Reply:
Seems like it's unnecessary, removed.